File: 2_Kernel/JavaKernel/src/main/java/com/seagosoft/tws/data/PnLSingleData.java
package com.seagosoft.tws.data;

import com.ib.client.Decimal;

public class PnLSingleData {

    public int reqId;
    public Decimal pos;
    public double dailyPnL;
    public double unrealizedPnL;
    public double realizedPnL;
    public double value;

    public PnLSingleData(int reqId, Decimal pos, double dailyPnL, double unrealizedPnL, double realizedPnL, double value) {
        this.reqId = reqId;
        this.pos = pos;
        this.dailyPnL = dailyPnL;
        this.unrealizedPnL = unrealizedPnL;
        this.realizedPnL = realizedPnL;
        this.value = value;
    }
}

File: 3_OrderSystem/KotlinEWrapperBackend/src/main/kotlin/com/seagosoft/tws/wrapper/structs/PnLSingleData.kt
package com.seagosoft.tws.wrapper.structs

import com.ib.client.Decimal


class PnLSingleData(
    var reqId: Int,
    var pos: Decimal,
    var dailyPnL: Double,
    var unrealizedPnL: Double,
    var realizedPnL: Double,
    var value: Double
)